_typos.toml

```diff
@@ -0,0 +1,7 @@
+
+[default]
+extend-ignore-re = ["(?Rm)^.*# spellchecker:disable-line$"]
+
+[default.extend-words]
+strat = "strat"
```

Comment:
Those two words are recognized as typos. As they occur quite often we whitelist them